server/src/test/python/cromwell-az-e2e-test/cromwell_az_e2e_test/az_e2e.py

```diff
@@ -0,0 +1,173 @@
+#       "attributes": {}}
+   
+#    create_workspace_response = requests.post(url=rawls_api_call, 
+#                                              json=request_body, 
+#                                              headers={"Authorization": f"Bearer {bearer_token}"}
+#    ).json()
+
+#    create_workspace_data = json.loads(json.dumps(create_workspace_response))
+#    workspaceId = create_workspace_data['workspaceId']
+
+#    print(f"Enabling CBAS for workspace {workspaceId}")
+#    activate_cbas_request = f"{leo_url}/api/apps/v2/{workspaceId}/terra-app-{str(uuid.uuid4())}"
+#    cbas_request_body = {
+#       "appType": "CROMWELL"
+#    } 
+        
+#    response = requests.post(url=activate_cbas_request, json=cbas_request_body, 
+#                             headers={"Authorization": f"Bearer {bearer_token}"})
+#    # will return 202 or error
+#    handle_failed_request(response, "Error activating CBAS", 202)
+   
+#    print(response)
+#    return workspaceId
+
+# # GET WDS OR CROMWELL ENDPOINT URL FROM LEO
+# def get_app_url(workspaceId, app):
+#     """"Get url for wds/cbas."""
+#     uri = f"{leo_url}/api/apps/v2/{workspaceId}?includeDeleted=false"
+
+#     headers = {"Authorization": bearer_token,
+#                "accept": "application/json"}
+
+#     response = requests.get(uri, headers=headers)
+#     status_code = response.status_code
+
+#     if status_code != 200:
+#         return response.text
+#     print("Successfully retrieved details.")
+#     response = response.json()
+
+#     app_url = ""
+#     app_type = "CROMWELL" if app != 'wds' else app.upper()
+#     print(f"App type: {app_type}")
+#     for entries in response: 
+#         if entries['appType'] == app_type and entries['proxyUrls'][app] is not None:
+#             print(entries['status'])
+#             if(entries['status'] == "PROVISIONING"):
+#                 print(f"{app} is still provisioning")
+#                 break
+#             print(f"App status: {entries['status']}")
+#             app_url = entries['proxyUrls'][app]
+#             break 
+
+#     if app_url is None: 
+#         print(f"{app} is missing in current workspace")
+#     else:
+#         print(f"{app} url: {app_url}")
+
+#     return app_url
+
+def submit_workflow_to_cromwell(app_url, workflow_test_name):
+    absolute_file_path = os.path.dirname(__file__)
+    workflow_source_path = os.path.join(absolute_file_path, 'workflow_files/hello.wdl')
+    workflow_inputs_path = os.path.join(absolute_file_path, 'workflow_files/hello.inputs')
+    workflow_endpoint = f'{app_url}/api/workflows/v1'
+    headers = {"Authorization": f'Bearer {bearer_token}',
+              "accept": "application/json",
+    }
+    files = {
+        'workflowSource': ('hello.wdl', open(workflow_source_path, 'rb'), 'application/octet-stream'),
+        'workflowInputs': ('hello.inputs', open(workflow_inputs_path, 'rb'), 'application/octet-stream'),
```

Comment:
In Python it's common to use a pattern like this, which will auto-close these open file handles when the block is done:
```
with open(path1, 'rb') as f1:
    with open(path2, 'rb') as f2:
        # make the request here
```